Dockerfile

```diff
@@ -8,8 +8,7 @@ RUN \
     cargo --version && \
     rustc --version && \
     mkdir -m 755 bin && \
-    cargo build --release && \
-    cp /app/target/release/autopush_rs /app/bin
+    cargo install --root /app
```

Comment:
I copied this from pushbox, so that prolly needs a similar fix.

Reply:
no fix here, just a 'refactor'. pushbox ran into a nice rust-nightly bug requiring this workaround for a few days

Reply:
I should check if that bug is still in play, TBH.

Reply:
I've verified it's fixed as of megaphone's pegged nightly-2018-05-05 or somewhere near there